Software/src/qtsingleapplication/common.pri

```diff
@@ -1,4 +1,4 @@
-exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtsingleapplication-uselib
+#exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtsingleapplication-uselib

```

Comment:
I'm not sure what this did but if it's really unnecessary please remove the line